src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -182,17 +182,26 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         # Run agent parameters
         c.argument('cpu', type=int, help='The CPU configuration in terms of number of cores required for the run.')
 
+        # MSI parameter
+        c.argument('assign_identity', nargs='*', help="Assigns managed identities to the task. Use '[system]' to refer to the system-assigned identity or a resource ID to refer to a user-assigned identity.")
```

Comment:
Using '*' implies that `--assign-identity` with no value is acceptable. Is this the case?  If not, use `nargs='+'`

Reply:
Yes, no value indicates to assign the system-assigned managed identity, which is the default managed identity type for a resource. The user can also enter the value `[system]` but we are treating this type of identity as the default. Azure VM supports a similar behavior https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-cli-windows-vm#system-assigned-managed-identity